### PR TITLE
A few minor fixes that the compiler picked up on.

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: iron-dropdown.html [hunk 1]
```diff
@@ -271,8 +272,7 @@
         },
 
         _cancelAnimations: function() {
-          this.cancelAnimation('open');
-          this.cancelAnimation('close');
+          this.cancelAnimation();
```
Comment (rictic) [inline iron-dropdown.html]: This is interesting, the playAnimation method takes an argument but cancelAnimation method doesn't: https://github.com/PolymerElements/neon-animation/blob/master/neon-animation-runner-behavior.html#L107

File: iron-dropdown.html [hunk 2]
```diff
@@ -239,7 +240,7 @@
           this.$.contentWrapper.classList.remove('animating');
           if (this.opened) {
             Polymer.IronOverlayBehaviorImpl._renderOpened.apply(this);
-          } else if (!this.cancelled) {
+          } else if (!this.canceled) {
```
Comment (cdata) [inline iron-dropdown.html]: Nice catch!

Comment (rictic) [inline iron-dropdown.html]: Yeah. Appeasing the compiler can be a pain in the ass, but it's nice when it catches this sort of thing.

File: iron-dropdown.html (hunk 1 above)
Comment (cdata) [inline iron-dropdown.html]: Yeah, that's kind of strange, but perhaps a simplification of cancel's implementation?

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**